x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/configured_sources_list.tsx

```diff
@@ -89,7 +89,7 @@ export const ConfiguredSourcesList: React.FC<ConfiguredSourcesProps> = ({
                 </EuiFlexItem>
                 {(!isOrganization || (isOrganization && !accountContextOnly)) && (
                   <EuiFlexItem grow={false}>
-                    <Link to={`${getSourcesPath(addPath, isOrganization)}/connect`}>
+                    <Link to={`${getSourcesPath(addPath, !!isOrganization)}/connect`}>
```

Comment:
Looks like `isOrganization` is `boolean | something-else`. Can we enforce `isOrganization` to be just `boolean` at the place where the variable is created?

Reply:
Yeah, the issue is that the value is optional in the component so it's possibly undefined.

Reply:
Fixed!